### PR TITLE
Homepage cohort list card changes

### LINKED ISSUE #51: Modify Cohort List component in the homepage
image

1) I think it's better to reverse the order of the cohorts : we would like to see the most recent cohorts first.
2) Also, remove for now the 2 columns "statut" and "favoris" which are not really useful

### PR DESCRIPTION
**Pull request by BPierrick**
## Fixes

- Fixes #51 

## Description

- Orders list of cohorts from most recent to older ones
- Removed `statut` & `favoris` columns

Capture d’écran 2021-03-31 à 16 47 42

### CODE REVIEW

File: src/services/savedResearches.ts [hunk 1]
```diff
@@ -132,6 +132,7 @@ export const fetchLastCohorts = async (): Promise<FormattedCohort[] | undefined>
             favorite: cohort.favorite
           }))
           .filter(Boolean)
+          .reverse()
```
Comment (Jasopaum) [inline src/services/savedResearches.ts]: Doesn't the ordering param in the query to cohort back work?

Comment (BPierrick) [reply src/services/savedResearches.ts]: I did not manage to reverse the ordering..

Param is like this `ordering=-created_at`

I tried `ordering=created_at` without success :(

Comment (simonvadee) [reply src/services/savedResearches.ts]: weird, it shoud work, I'll take a look at the backend

**Comment by MiskoG** [comment]
For me UI-wise it's perfect 👍

File: src/services/savedResearches.ts (hunk 1 above)
Comment (simonvadee) [reply src/services/savedResearches.ts]: fixed

**Review by simonvadee — APPROVED** [review]

**Pull request merged**